src/main/java/org/takes/facets/auth/social/PsGithub.java

```diff
@@ -66,13 +67,30 @@
     private final transient String key;
 
     /**
+     * Urls replacements.
+     */
+    private final transient Map<String, String> substs;
+
+    /**
      * Ctor.
      * @param gapp Github app
      * @param gkey Github key
      */
     public PsGithub(final String gapp, final String gkey) {
+      this(gapp, gkey, Collections.<String, String>emptyMap());
+    }
+
+    /**
+     * Ctor.
+     * @param gapp Github app
+     * @param gkey Github key
+     * @param urls Urls replacements
+     */
+    PsGithub(final String gapp, final String gkey,
+        final Map<String, String> urls) {
         this.app = gapp;
         this.key = gkey;
+        this.substs = new ConcurrentHashMap<String, String>(urls);
```

Comment:
@dmzaytsev What is this "substitution map" for? It doesn't look like the same constructor being requested by @yegor256 in #42.